rp_asos/models.py

```diff
@@ -71,6 +73,41 @@ def add_group_admins(self, group_info, wa_ids):
                     self.project.org, group_info["id"], wa_id
                 )
 
+    def send_message(self, reminders):
+        group_info = self.get_wa_group_info()
+        rapidpro_client = self.project.org.get_rapidpro_client()
+
+        urns = {self.hospital_lead_urn: self.hospital_lead_name}
+        if self.nomination_urn:
+            urns[self.nomination_urn] = self.nomination_name
+
+        sent_to_group = False
+        for urn, name in urns.items():
+            if urn.replace("+", "") in group_info["participants"]:
+                if not sent_to_group:
+                    sent_to_group = True
+
+                    utils.send_whatsapp_group_message(
```

Comment:
I'm not sure how we're handling potential failures here. We could either inspect the response here or we could bake this in to the `send_whatsapp_group_message` method. We could use `raise_for_status` so that we get a sentry error or set a retry? My thinking is that we could retry 3 times in cases of rate-limiting, and raise an alert after that.

Reply:
Another way that we could deal with this, could be to move the calls to Turn to a client Class and then create a retry and raise for failure functionality that all of the calls use? That might be more work than we have capacity for, let me know if you need help with that.

Reply:
I think some retries with `raise_for_status` will have to do for now.